### ISSUE TITLE
Axe - Does not report duplicate failures when more than 1 failure exists for a specific metric

### ISSUE THREAD

**Issue by luke-hill** [pa11y/pa11y#506]
---

## Expected behaviour
Pa11y reports **multiple failures** on Axe metric:  • Error: Elements must have sufficient color contrast (https://dequeuniversity.com/rules/axe/3.4/color-contrast?application=axeAPI)
```
Welcome to Pa11y

Running Pa11y on URL BLAH

Results for URL: BLAH

 • Error: ARIA hidden element must not contain focusable elements (https://dequeuniversity.com/rules/axe/3.4/aria-hidden-focus?application=axeAPI)
   ├── aria-hidden-focus
   ├── BLAH 1
   └── <li aria-hidden="true" ...</li>

 • Error: Elements must have sufficient color contrast (https://dequeuniversity.com/rules/axe/3.4/color-contrast?application=axeAPI)
   ├── color-contrast
   ├── BLAH
   └── <p>text</p>

 • Error: Elements must have sufficient color contrast (https://dequeuniversity.com/rules/axe/3.4/color-contrast?application=axeAPI)
   ├── color-contrast
   ├── BLAH2
   └── <p>text2</p>

 • Error: Elements must have sufficient color contrast (https://dequeuniversity.com/rules/axe/3.4/color-contrast?application=axeAPI)
   ├── color-contrast
   ├── BLAH3
   └── <p>text3</p>

4 Errors
```
## Actual behaviour
Pa11y reports **1 failure** on Axe metric:  • Error: Elements must have sufficient color contrast (https://dequeuniversity.com/rules/axe/3.4/color-contrast?application=axeAPI)
```
Welcome to Pa11y

Running Pa11y on URL BLAH

Results for URL: BLAH

 • Error: ARIA hidden element must not contain focusable elements (https://dequeuniversity.com/rules/axe/3.4/aria-hidden-focus?application=axeAPI)
   ├── aria-hidden-focus
   ├── BLAH 1
   └── <li aria-hidden="true" ...</li>

 • Error: Elements must have sufficient color contrast (https://dequeuniversity.com/rules/axe/3.4/color-contrast?application=axeAPI)
   ├── color-contrast
   ├── BLAH
   └── <p>text</p>

2 Errors
```
## Steps to reproduce
You need a site with multiple failures on this metric (Or possibly others)
`$ pa11y --runner axe url_of_multiple_failures`

## Environment:
```
$ pa11y --environment
Pa11y:      5.3.0
Node.js:    12.14.1
npm:        6.13.4
OS:         4.15.0-76-generic (linux)
```

**Comment by josebolos** [comment]
Hi @luke-hill,

Thanks for creating this issue! Could you please include an example of the incorrect output? It's not very clear from the issue description what the problem is.

**Comment by luke-hill** [comment]
@josebolos Added in just now.

**Comment by luke-hill** [comment]
@josebolos any news on this? Can you reproduce?

**Comment by josebolos** [comment]
Sure! It seems like HTML CodeSniffer and axe report information differently, and this wasn't accounted for when integrating axe with pa11y. It definitely should be addressed.

**Comment by joeyciechanowicz** [comment]
Closing as this is a duplicate on the `pa11y-runner-axe` where this behaviour stems from.
https://github.com/pa11y/pa11y-runner-axe/issues/9
